### PR TITLE
Wait for encode to finish. Part of avalonmediasystem/avalon#1827

### PR DESCRIPTION
**Pull request by val99erie**
Here are some notes about how to run an encoding job on AWS.  We can add some similar notes on the `active_encode_dev_con` branch once we're finished with the code and the branch is ready to merge into master.
https://github.com/curationexperts/active-encode-demo/blob/master/doc/transcoding_with_amazon.md

### CODE REVIEW

**Comment by jcoyne** [comment]
You may want to be specific and say these directions are for ec2 (right?)

**Comment by val99erie** [comment]
@jcoyne The file encoding happens in the Elastic Transcode pipeline/jobs in AWS, but the instructions for triggering the job in the rails console with the demo app, I just run on my mac laptop.  Is that what you meant?

**Comment by jcoyne** [comment]
I see. I thought you were running part of the app on EC2. In that case I'd just change "AWS Pipeline" to "Elastic Transcode Pipeline", so as not to confuse it with Code Pipeline which is also in AWS.

**Comment by val99erie** [comment]
@jcoyne Ok will do.  Thanks!

File: lib/hydra/derivatives/runners/active_encode_derivatives.rb [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-    # @options options [Array] :outputs a list of desired outputs, each entry is a hash that has :label (optional), :format and :url
+    # @options options [Array] :outputs a list of desired outputs
     def self.create(object_or_filename, options)
-      source_file(object_or_filename, options) do |f|
-        instructions = transform_directives(options)
-        processor_class.new(f,
-                            instructions.merge(source_file_service: source_file_service),
-                            output_file_service: output_file_service).process
+      file_name = object_or_filename
+      transform_directives(options.delete(:outputs)).each do |instructions|
```
Comment (cjcolvar) [inline lib/hydra/derivatives/runners/active_encode_derivatives.rb]: I'm not quite understanding the reason to switch from all H-D directives being run by a single processor instance to each H-D directive getting its own processor.  Is there a case where one processor can't handle all of the directives at once?  Another way to ask this is is there any case where a call to ActiveEncodeDerivatives.create should spawn multiple ActiveEncode::Base.create calls?

**Pull request merged**